Integrations/integration-OpsGenie.yml

```diff
@@ -93,27 +100,65 @@ script:
         var res = sendRequest(url);
         var md = '### OpsGenie User Info\n';
         var ec = {};
-        if (res && res.data && res.data) {
-            var user = res.data;
-            md += objToMd(user);
+        if (res && res.data) {
+            md += objToMd(res.data);
         } else {
             md = 'Cannot find user ' + userID + '\n';
         }
         return {Type: entryTypes.note, Contents: res.data, EntryContext: ec, ContentsFormat: formats.json, HumanReadable: md};
     };
 
+    var getSchedules = function() {
+        var url = serviceURLs.schedules;
+        var res = sendRequest(url);
+        var md = '### OpsGenie On Call Schedules\n';
+        var ec = {};
+        if (res && res.data) {
+            md += tableToMarkdown('',res.data,['name','timezone','enabled','description']);
+        } else {
+            md = 'No schedules\n';
+        }
+        return {Type: entryTypes.note, Contents: res.data, EntryContext: ec, ContentsFormat: formats.json, HumanReadable: md};
+    };
+
+    var getScheduleTimeline = function(schedule) {
+        var url = serviceURLs.schedules + '/' + schedule + '/timeline?identifierType=name';
+        var res = sendRequest(url);
+        var rotations = res.data.finalTimeline.rotations;
```

Comment:
you are checking below for `(res && res.data)` but you access `res.data` without checking.. you might have issue here. 
Just move this line inside the if statement

Reply:
yep... fixed.